roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/ds-ingress-nginx-controller.yml.j2

```diff
@@ -51,9 +51,8 @@ spec:
             - --tcp-services-configmap=$(POD_NAMESPACE)/tcp-services
             - --udp-services-configmap=$(POD_NAMESPACE)/udp-services
             - --annotations-prefix=nginx.ingress.kubernetes.io
-{% if ingress_nginx_class is defined %}
             - --ingress-class={{ ingress_nginx_class }}
```

Comment:
Is it fine to specify this `--ingress-class` option always even if disabling the nginx class?

Reply:
It is no longer possible to disable the `ingressClass`, as it will always be deployed so it makes sense to set this options as well

Reply:
Thank you for your explanation, I got it.
The question came from the part of role-ingress-nginx.yml.j2 which contained the condition still.
But the condition has been removed already.
So looks good for me now.